toxcore/net_profile.c

```diff
@@ -0,0 +1,117 @@
+/* SPDX-License-Identifier: GPL-3.0-or-later
+ * Copyright © 2022 The TokTok team.
+ */
+
+/*
+ * Functions for the network profile.
+ */
+
+#include "net_profile.h"
+#include "ccompat.h"
+
+#define NETPROF_TCP_DATA_PACKET_ID 0x10
+
+/* Returns the number of sent or received packets for all ID's between `start_id` and `end_id`. */
```

Comment:
Prefer doxygen style comments (`/**`). Those are checked a bit more than regular comments. (No need to add more documentation here, just change the comment style)